Stylesheets/components/input.css

```diff
@@ -0,0 +1,75 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap');
+
+body{
+    font-family: 'Montserrat', sans-serif;
+    
+}
+
+.flex{
+    display: flex;
+    flex-direction: column;
+    width:10rem;
+    margin-bottom: 1rem;
+    width:30rem
```

Comment:
You have mentioned width twice here. If you intended max width here , please correct that here.